docs/adguard-for-android/solving-problems/low-level-settings.md

```diff
@@ -184,27 +188,27 @@ Here you can set the maximum transmission unit (MTU) of the VPN interface. The r
 
 #### Restore VPN automatically
 
-If enabled, this feature automatically re-enables AdGuard’s local VPN after it has been turned off due to network absence, tethering, or low-power mode.
+This option automatically re-enables AdGuard’s local VPN after it has been turned off due to network absence, tethering, or low-power mode.
```

Comment:
If this is enabled, AdGuard’s local VPN will be automatically re-enabled after being turned off due to network absence, tethering, or low-power mode.